Packs/Core/Playbooks/Playbook-Large_Upload_Alert.yml

```diff
@@ -0,0 +1,3959 @@
+      script: Builtin|||setAlertStatus
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      status:
+        simple: Resolved - False Positive
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 890,
+          "y": 4015
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "58":
+    id: "58"
+    taskid: df644331-858b-45bc-8eb8-7a619e09aa7f
+    type: regular
+    task:
+      id: df644331-858b-45bc-8eb8-7a619e09aa7f
+      version: -1
+      name: Close Alert - Resolve As True Positive
+      description: commands.local.cmd.set.alert.status
+      script: Builtin|||setAlertStatus
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      status:
+        simple: Resolved - True Positive
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 0,
+          "y": 4940
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "59":
+    id: "59"
+    taskid: b7d7224b-61d5-4130-8e49-eda6fad4eef4
+    type: regular
+    task:
+      id: b7d7224b-61d5-4130-8e49-eda6fad4eef4
+      version: -1
+      name: Close Alert - Resolve As Other
+      description: commands.local.cmd.set.alert.status
+      script: Builtin|||setAlertStatus
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      status:
+        simple: Resolved - Other
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1490,
+          "y": 1110
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "14_20_#default#": 0.5,
+      "17_20_#default#": 0.16,
+      "19_20_#default#": 0.11,
+      "22_23_yes": 0.46,
+      "27_34_#default#": 0.28,
+      "31_34_#default#": 0.21,
+      "32_34_#default#": 0.25,
+      "39_46_Remediation": 0.18,
+      "39_57_#default#": 0.31,
+      "3_6_#default#": 0.34,
+      "48_58_#default#": 0.58,
+      "4_56_#default#": 0.41,
+      "5_56_#default#": 0.43,
+      "5_6_Yes": 0.13
+    },
+    "paper": {
+      "dimensions": {
+        "height": 5305,
+        "width": 2640,
+        "x": -550,
+        "y": -130
+      }
+    }
+  }
+inputs:
+- key: InternalIPRanges
+  value:
+    simple: 172.16.0.0/12,10.0.0.0/8,192.168.0.0/16
```

Comment:
Should be a list instead of this range hardcoded.

Reply:
changed